### PR TITLE
Parameter decode

### PR DESCRIPTION
**Pull request by tbaik**

### CODE REVIEW

File: src/com/tony/server/response/DirectoryResponse.java [hunk 1]
```diff
@@ -1,7 +1,6 @@
 package com.tony.server.response;
 
 import java.util.ArrayList;
-
```
Comment (jayden) [inline src/com/tony/server/response/DirectoryResponse.java]: why did you get rid of this space?

File: src/com/tony/server/WorkerThread.java [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 import com.tony.server.response.Response;
 
-import java.io.*;
+import java.io.IOException;
+import java.io.InputStream;
```
Comment (jayden) [inline src/com/tony/server/WorkerThread.java]: `N` before `O` bro

File: src/com/tony/server/ResponseDeterminer.java [hunk 3]
```diff
@@ -28,6 +29,8 @@ public Response determineResponse(Request request) {
             }
         } else if(uriList.contains(request.getURI())){
             return new MethodNotAllowedResponse();
+        } else if(request.getURI().startsWith("/parameters?")){
```
Comment (jayden) [inline src/com/tony/server/ResponseDeterminer.java]: what happens when a request comes in at  `/parameter` ? do you issue a 404? I feel like you shouldn't be checking to see if the `?` exists here.

Comment (tbaik) [inline src/com/tony/server/ResponseDeterminer.java]: Sorry, I meant to work on the parameter decode changes in this branch but I did it in the branch 1 ahead of this one. https://github.com/tbaik/java-server/commit/d07ddfc79a621f584cf14125a0d4127e1edcef64#diff-bf58bc1910ee85f9e4946f490ba5410aR40

**Pull request merged**